Update examples and add one for cross account (#70)
DataDog/terraform-module-datadog-agentless-scanner: examples/README.md

# Single Region Example

> With this option, a single Agentless scanner is deployed in a single region. Although this can incur more costs, as it requires each Agentless scanner to perform cross-region scans per account, Datadog recommends this option.

Reviewer: ```suggestion
With this option, a single Agentless scanner is deployed in a single region. Datadog recommends this option. However, this option can incur higher costs, as it requires each Agentless scanner to perform cross-region scans per account.
```

> To deploy in a single regions you can check this [example](multi_region/README.md).

Reviewer: ```suggestion
To deploy in a single region, see the [example](multi_region/README.md).
```

# Multi Region Example

With this option, Agentless scanners are deployed on a single cloud account and are distributed across multiple regions within the account. With this deployment model, Agentless scanners are granted visibility without needing to perform cross-region scans, which are expensive in practice.

> To deploy in multiple regions you can check this [example](multi_region/README.md).

Reviewer: ```suggestion
To deploy in multiple regions, see the [example](multi_region/README.md).
```

# Cross Account Example

> With this option, Agentless scanner is deployed in a single or multi-region set-up on a single account.

Reviewer: ```suggestion
With this option, an Agentless scanner is deployed in a single or multi-region set-up on a single account.
```

> You don't need to deploy the scanner in any of you other accounts, instead a single delegate role will be created that will allows the scanner access to that account.

Reviewer: ```suggestion
You don't need to deploy the scanner in any of your other accounts. Instead, a single delegate role is created that allows the scanner access to that account.
```

> If you are interested to scan your other accounts you can check that [example](cross_account/README.md)

Reviewer: ```suggestion
To scan across accounts, see the [example](cross_account/README.md)
```

# Custom VPC Example

> If for any reasons you want to avoid creating a new VPC for the Agentless scanners and want to re-use one of your own you can check that [example](custom_vpc/README.md)

Reviewer: ```suggestion
If you want to avoid creating a new VPC for the Agentless scanners, and you want to reuse one of your own, see the [example](custom_vpc/README.md)
```
